NEWS

```diff
@@ -68,6 +68,12 @@ Major new features:
 Changes in behavior compared to prior versions:
 
 Removed legacy support:
+- "yoda" SPML component is removed from the Open SHMEM codebase. It was utilizing
+  Open MPI BTL components for supported networks.
+  Currently avalable SPML components are:
```

Comment:
I'd rather this just include the comment about yoda removal, not the other items.  It makes the removed legacy support section harder to read.